.env.template

```diff
@@ -5,3 +5,4 @@ SOPHIE_AUTHORIZATION=#{SOPHIE_AUTHORIZATION}
 SOPHIE_TENANT=#{SOPHIE_TENANT}
 YOUTUBE_API_KEY=#{YOUTUBE_API_KEY}
 VERSION_DEPLOYED=#{VERSION_DEPLOYED}
+SITE_URL=#{SITE_URL}
```

Comment:
maybe leave this out for now since it will break the build and we don't need to override it just yet

Reply:
variable change from the environment file to site-config